components/bl-copyButton-component/component.json

```diff
@@ -0,0 +1,68 @@
+{
+  "id": "c_085c78f4c41233294eed6707cb128d5d",
+  "name": "Copy Button",
+  "description": "This component allows add copy to clipboard button",
+  "showInToolbox": true,
+  "faIcon": "copy",
+  "mainJS": "dist/index.js",
+  "type": "custom",
+  "category": "Custom Components",
+  "properties": [
+    {
+      "type": "checkbox",
+      "name": "isDisabled",
+      "label": "Disabled",
+      "handlerId": "isDisabledLogic",
+      "handlerLabel": "Disabled Logic",
+      "handlerDescription": "This is a handler for the logic to determine disabled or not disabled button"
+    },
+    {
+      "name": "content",
+      "label": "Content",
+      "handlerId": "contentLogic",
+      "handlerLabel": "Content Logic",
+      "dataBinding": true,
+      "type": "text",
+      "handlerDescription": "This is the handler for the logic of determining the content to be copied."
+    },
+    {
+      "type": "text",
+      "name": "copyLabel",
+      "label": "Copy Label",
```

Comment:
Could we also add Copy Icon with Icon property type? Same for copied icon

Reply:
Or we could make different types: `default` | `icon` | `complex`(text and icon in same time)